style: Add styles to greeter for not-logged-in users according to Figma mockup.

sultanowski opened the pull request:
### What
- [x] Added styling according to Figma mockup.
- [x] Added partially LTR support.
- [x] Removed not used conditions in `.../templates/site_layout.tt.html`
- [x] Removed spotted empty style block in `~/scss/_off.scss`
- [x] CSS LSP undescored that `font-ligature-setting` is provided only for WebKit browsers, so I added a line to remove this warning.
- [x] Commented out "Pernament link to results..." since this feature is not working anyway and it does not fit in this place anymore.

stephanegigandet commented:
The first page when not logged in looks very good!

![image](https://user-images.githubusercontent.com/8158668/213657980-ec717eec-8bdd-476e-8776-bd13f0c5a044.png)

stephanegigandet commented:
On medium screens, the text over the illustrations is not completely legible (even if the illustrations are slightly lightened).

![image](https://user-images.githubusercontent.com/8158668/213658462-5f1c2eac-6abb-42e8-bba8-42d4b3d3c6b3.png)

stephanegigandet commented on templates/web/common/site_layout.tt.html:
```diff
@@ -249,21 +249,9 @@ <h1 itemprop="name">[% title %]
-								</div>
 								[% content %]
 							[% END %]
-						[% ELSE %]
```
This part should not be removed, otherwise it makes all content from pages that are not product pages, list of products and product edit form have not content.

e.g. the "Discover" page: 
![image](https://user-images.githubusercontent.com/8158668/213659044-babbd168-b23b-4fa2-bbce-ff05f6375cc1.png)

stephanegigandet commented on templates/web/common/site_layout.tt.html:
```diff
@@ -249,21 +249,9 @@ <h1 itemprop="name">[% title %]
 									[% content %]
 								</div>
 							[% ELSE %]
-								<div class="row">
```
This should be kept, if there is no schema_org_itemtype defined. (today there's always the "Thing" schema by default, but that may change).

stephanegigandet commented:
When signed-in, there's an issue with the background colors:

![image](https://user-images.githubusercontent.com/8158668/213660737-6300fea2-c8c8-4bea-8582-2b20ab84a75e.png)

stephanegigandet reviewed (changes requested):
The greeter looks very good, but it broke a few things.

sultanowski commented:
I am going to resolve those issues ASAP. Thansks for review!

sultanowski commented:
- [x] Reverted changes to `site-layout.tt.html` file - it renders content on other pages.
- [x] Added global class to not display HTML tags without content.
- [x] Changed opacity of images to lower on smaller devices to make texts more visible.
- [x] Fixed issues with full-height background color.

stephanegigandet reviewed (approved):
I tested it and it looks very good, thanks for making the changes @sultanowski!
